src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -878,25 +878,15 @@ type SynExpr =
     /// F# syntax: let f pat1 .. patN = expr in expr
     /// F# syntax: let rec f pat1 .. patN = expr in expr
     /// F# syntax: use pat = expr in expr
-    | LetOrUse of
-        isRecursive: bool *
-        isUse: bool *
-        bindings: SynBinding list *
-        body: SynExpr *
-        range: range *
-        trivia: SynExprLetOrUseTrivia
-
     /// F# syntax: let! pat = expr in expr
     /// F# syntax: use! pat = expr in expr
     /// F# syntax: let! pat = expr and! ... and! ... and! pat = expr in expr
-    /// Computation expressions only
-    | LetOrUseBang of
-        bindDebugPoint: DebugPointAtBinding *
+    | LetOrUse of
+        isRecursive: bool *
         isUse: bool *
         isFromSource: bool *
```

Comment:
isFromSource likely also deserves a comment - what does it mean when its `true` for a regular `let` ?

Reply:
Added a comment to the fsi file